.github/dependabot.yaml

```diff
@@ -0,0 +1,39 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/drivers/
```

Comment:
We don't want the driver images to be auto-updated, that can lead to driver errors and would be pretty bad for us. Please remove these.